openquake/hazardlib/source/rupture.py

```diff
@@ -231,7 +231,8 @@ class ParametricProbabilisticRupture(BaseProbabilisticRupture):
         If occurrence rate is not positive.
     """
     __slots__ = Rupture.__slots__ + [
-        'occurrence_rate', 'temporal_occurrence_model']
+        'occurrence_rate', 'temporal_occurrence_model',
+        'rupture_slip_direction']
```

Comment:
Modified. Now the slot is in Rupture class. It can be used both in ParametricProbabilisticRupture, and NonParametricProbabilisticRupture now.